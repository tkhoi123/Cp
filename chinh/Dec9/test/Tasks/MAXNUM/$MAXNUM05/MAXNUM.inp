8
806 676 107 490 745 507 67 831 