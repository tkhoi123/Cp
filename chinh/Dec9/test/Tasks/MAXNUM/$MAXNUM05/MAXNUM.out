83180674567676507490107